src/pages/_partials/guide.ejs

```diff
@@ -57,7 +57,7 @@ var isSideNavEntrySelected = function(entry, i) {
 
 <main class="mdl-layout__content">
     <header class="hero-background">
-        <section class="section mdl-grid mdl-grid--no-spacing">
```

Comment:
let's add `--no-spacing`, `h4` already has `margin: 40px 0;`, otherwise the hero will have additional padding.